build.sh

```diff
@@ -193,30 +223,38 @@ if [ "$xSTATIC_ANALYSIS" = "1" ] ; then
             ;;
         esac
 
-        if ! pep8_result=`pep8 --first --ignore=$PEP8_IGNORES $f` ; then
-          echo "$pep8_result"
-
+        if ! pep8_result=`pep8 --ignore=$PEP8_IGNORES $f` ; then
+          printf '%s\n' "$pep8_result" | while IFS= read -r line
+          do
+            echo "MSGBLD0195: PEP8 $line"
+          done
           format_error_files="$format_error_files $f"
         fi
-        echo "======= Check PEP8 end ======="
+        echo "MSGBLD0200: PEP8 check for file $f completed."
         ;;
       *)
-        echo "$f : not a C/CPP/PY file. Do not do static analysis / formatting checking."
+        echo "MSGBLD0210: File $f is not a C/CPP/PY file. Static code analysis and formatting check skipped."
```

Comment:
Change `CPP` to `C++` and `PY` to `Python` so that the sentence refers to languages rather than file name extensions.